.github/workflows/release-minor.yml

```diff
@@ -0,0 +1,28 @@
+name: Release minor Workflow
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run && github.event.workflow_run.conclusion == 'success' || true }}
+    environment: 'Release NPM'
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      GIT_USER: ${{ vars.GIT_USER }}
+      GIT_EMAIL: ${{ vars.GIT_EMAIL }}
+      RELEASE_BRANCH: ${{ github.event.workflow_run.head_branch || github.ref }}
+    steps:
+      - uses: ./.github/release/preparation
+      - name: Build packages
+        run: npx nx run-many --target=build --all --parallel=2 --exclude storefront,fulfillment
+      - name: Fix js extension
+        run: node tools/post-build/add-js-extension.js
+      - name: Release patch
+        if: ${{ startsWith(env.RELEASE_BRANCH, 'refs/heads/release/') }}
+        run: npm run minor
+      - name: Release Error Log
+        if: failure()
+        run: cat libs/lerna-debug.log
```

Comment:
Is it possible that sensitive info (aka keys) will be logged and visible to the public?

Reply:
Only for someone who has an access to ci.